core/src/test/java/org/elasticsearch/search/aggregations/bucket/SamplerIT.java

```diff
@@ -22,6 +22,7 @@
 import org.elasticsearch.action.search.SearchResponse;
 import org.elasticsearch.action.search.SearchType;
 import org.elasticsearch.index.query.TermQueryBuilder;
+import org.elasticsearch.search.aggregations.bucket.sampler.DiversifiedSamplerAggregationBuilder;
```

Comment:
can you split it into a file for the regular sampler and another one for the diversified sampler?